lib/artifact-search.js

```diff
@@ -68,28 +78,38 @@ module.exports = (bluebird, error, logger, requestOptions, requestPromise, respo
 
 
         /**
+         * @typedef {Object} searchParameters
+         * @property {(string|string[])} search A string or array of strings
+         *                                      for the search criteria. Can be
+         *                                      undefined.
+         * @property {(string|string[])} sort A string or array of strings for
+         *                                    how the results will be sorted.
+         * @property {number} limit A number denoting how many results should
+         *                          be returned from the search.
+         */
+
+        /**
```

Comment:
According to the style guide JSDoc blocks should be padded by 2 newlines :-( - "Blank line above comments, two blank lines above JSDoc comments." I only see two instances of this. Not that I really care much.